app/views/layouts/_common_ngapp.html.haml

```diff
@@ -0,0 +1,37 @@
+= render :partial => 'layouts/doctype'
```

Comment:
Hm, why is this layout needed, can't we just use the default one (I mean `application.html.haml`)?